README.md

```diff
@@ -5,9 +5,7 @@ It includes the rollup and fraud proof smart contracts, as well as interfaces fo
 
 For more information see https://developer.arbitrum.io/intro
 
-For the deployed addresses of these contracts for Arbitrum chains see https://developer.arbitrum.io/useful-addresses
-
-For the token bridge contracts see https://github.com/OffchainLabs/token-bridge-contracts
```

Comment:
I think we might want to leave this line since it's helpful to link people to that other repo for discovery

Reply:
reinstated the token bridge contracts repo link!